src/glue/js/binaryen.d.ts

```diff
@@ -10,3 +10,8 @@ declare namespace binaryen {
   }
   function wrapModule(ptr: number): Module;
 }
+
+/** Stores an 8-bit value to Binaryen memory. */
+declare function store<T extends u8>(ptr: usize, value: T): void;
+/** Loads an 8-bit value from Binaryen memory. */
+declare function load<T extends u8>(ptr: usize): T;
```

Comment:
These are somewhat awkward still. Ideally, Binaryen would expose a `_store8`, `_store32` etc. along `_malloc` and friends so we don't need any JS glue when importing and interacting with Binaryen's memory.

Reply:
There is [this Binaryen PR](https://github.com/WebAssembly/binaryen/pull/2476) now suggesting a set of Binaryen exports named after the respective Wasm instructions so we can get rid of the store and load stuff as well. Instead of mixing in suboptimal functions, we'd then be able to simply call Binaryen's `_i32_store8` just like we call `_malloc`.